priv/template/lib/application_name/endpoint.ex

```diff
@@ -4,7 +4,8 @@ defmodule <%= application_module %>.Endpoint do
   # Serve at "/" the given assets from "priv/static" directory
   plug Plug.Static,
     at: "/", from: :<%= application_name %>,
-    only: ~w(css images js favicon.ico robots.txt)
+    only: ~w(css images js favicon.ico robots.txt
+             app.js app.js.map app.css app.css.map)
```

Comment:
Ah, this would be a reason to not compile stuff to root. Let's change the brunch file?